pandas/conftest.py

```diff
@@ -126,6 +126,12 @@ def observed(request):
     return request.param
 
 
+@pytest.fixture(params=[True, False, None])
+def ordered(request):
```

Comment:
I think this could be more generic than `ordered` since it would have applications for other parameters. Maybe something like `optional_bool` would be more fitting?

Reply:
I see your point, but there are also benefits of naming the fixture after the parameter being parametrized and not sharing. For example, if e.g. if 10 parametrizations would share a fixture and then one would change its signature, it would be more difficult to find where its fixture is used.

Reply:
I agree with @topper-123 here.  This is fine.

Reply:
this is consistent as it’s naming is specific to its use; could name it ordered_fixture though as we use this particular nomenclature